en/reference/schema-reference.html

```diff
@@ -3374,11 +3366,68 @@ <h3 id="changes-that-require-re-feed">Changes that require re-feed</h3>
         This changes the document processing pipeline before documents
         arrive in the backend, and what annotations are made for an indexed field.
         <br>
-        NOTE: If not re-feeding after such a change, serving works,
+        NOTE: If not reindexing after such a change, serving works,
         but recall is undefined as the index has been produced using one match mode
         while run-time is using a different match mode
       </td></tr>
 
+  </tbody>
+</table>
+Example: Given a content cluster <em>mycluster</em> with mode <em>index</em>:
```

Comment:
This is a reference, ideally we should write a guide doc whenever we feel a need to add lots of explanation like this.